### ISSUE TITLE
ADE20K validation accuracy

### ISSUE THREAD

**Issue by hkchengrex** [zhanghang1989/PyTorch-Encoding#234]
Using this command
`python test.py --backbone resnet101 --base-size 640 --crop-size 576 --model-zoo EncNet_ResNet101_ADE --eval`
gives me 44.31% mIoU as opposed to 44.4% reported [here](https://hangzhang.org/PyTorch-Encoding/experiments/segmentation.html) or 44.65% in the paper.

Am I missing anything here? Thanks!
